tests/test-cases/Group6-VIC-Machine/6-10-List.robot

```diff
@@ -54,4 +54,8 @@ List with trailing slash
     Should Contain  ${ret}  ID
     Should Contain  ${ret}  PATH
     Should Contain  ${ret}  NAME
-    Should Not Contain  ${ret}  Error
\ No newline at end of file
+    Should Not Contain  ${ret}  Error
+    
+List suggest compute resource
+    ${ret}=  Run  bin/vic-machine-linux ls --target %{TEST_URL} --thumbprint=%{TEST_THUMBPRINT} --user %{TEST_USERNAME} --password=%{TEST_PASSWORD} --compute-resource fakeComputeResource
+    Should Contain  ${ret}  Suggested values for --compute-resource:
```

Comment:
it is nice. Please also add another case to test list function happy path, I mean with correct dc, and no compute resource specified